README.md

```diff
@@ -24,17 +24,17 @@ This module provides Cloud Trace support for Node.js applications. [Google Cloud
 
         npm install --save @google/cloud-trace
 
-2. Include and start the library at the *top of the main script of your application*. It's important that Cloud Trace is the first thing executed so that it can accurately gather data:
+2. If you are running locally, or not on the Google Cloud Platform, set the GCLOUD_PROJECT environment variable. Your project ID is visible in the [Google Cloud Console Console](https://console.cloud.google.com/project?_ga=1.258049870.576536942.1443543237), it may be something like `particular-future-12345`. If your application is [running on Google Cloud Platform](running-on-google-cloud-platform), you don't need to specify the project ID.
 
-        require('@google/cloud-trace').start({projectId: 'your-project-id'});
+3. Include and start the library at the *top of the main script of your application*. If you use `--require` in your start up command, make sure that the trace agent is --required first. It's important that Cloud Trace is the first thing executed so that it can accurately gather data:
```

Comment:
If you're feeling really nice, show an example.  If you're feeling super nice, warn people that it's `bin/www` in the case of an express app :)